packages/expo-store-review/android/src/main/java/expo/modules/storereview/StoreReviewModule.kt

```diff
@@ -6,65 +6,63 @@ import android.os.Build
 import com.google.android.gms.common.GooglePlayServicesUtil
 import com.google.android.play.core.review.ReviewManager
 import com.google.android.play.core.review.ReviewManagerFactory
-import expo.modules.core.ExportedModule
-import expo.modules.core.ModuleRegistry
-import expo.modules.core.Promise
-import expo.modules.core.interfaces.ActivityProvider
-import expo.modules.core.interfaces.ExpoMethod
+import expo.modules.kotlin.Promise
+import expo.modules.kotlin.functions.Queues
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
 
-class StoreReviewModule(private val mContext: Context) :
-  ExportedModule(mContext) {
-  companion object {
-    private const val NAME = "ExpoStoreReview"
-  }
+class StoreReviewModule : Module() {
+  private val context: Context
+    get() = requireNotNull(appContext.reactContext) { "React Application Context is null" }
```

Comment:
```suggestion
  private val context: Context
    get() = appContext.reactContext ?: throw Exceptions.ReactContextLost()
```
You can use https://github.com/expo/expo/blob/4b3b7a17df0bb2810bbb59d3af34b22b548cb18c/packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/exception/CommonExceptions.kt#L20 instead of doing `requireNotNull`